common/app/views/fragments/email/signup/emailSignUp.scala.html

```diff
@@ -29,6 +29,7 @@
             </div>
             <button type="submit" class="email-sub__submit-input js-email-sub__submit-input button button--tertiary button--large">@fragments.inlineSvg("envelope", "icon", Seq("submit-input__icon"))@submitText</button>
             <small class="email-sub__small js-email-sub__small">@smallText</small>
+            <label class="email-sub__small email-sub__small--mark email-sub__small--is-hidden js-email-sub__small--mark"><input type="checkbox" name="mark" class="email-sub__mark-input js-email-sub__mark-input"/></label>
```

Comment:
what if there's no css? if it's only used in a JS context, could you insert it with js?